outputs.tf

```diff
@@ -220,3 +220,51 @@ output "workspace_bucket_name" {
   value       = module.s3.workspace_bucket_name
   description = "ID of the S3 bucket used for storing workspaces."
 }
+
+output "shell" {
+  sensitive = true
```

Comment:
this one isn't sensitive imo, the other one `tfvars` is.

Reply:
Yeah - it's just like that because originally I had sensitive values in it. I can change it, although it doesn't do any harm.